test/test_metric/test_metrics.py

```diff
@@ -116,7 +118,7 @@ def test_proba_scorer_binary(self):
         y_pred = [[1.0, 0.0], [1.0, 0.0], [0.0, 1.0], [0.0, 1.0]]
 
         scorer = autosklearn.metrics._ProbaScorer(
-            'accuracy', sklearn.metrics.log_loss, 0, 1, {})
+            'accuracy', sklearn.metrics.log_loss, 0, MAXINT, 1, {})
```

Comment:
Actually, as you're touching these lines, could you please change the names of the scorers?